src/Services/PaymentCardTokenService.php

```diff
@@ -46,4 +65,17 @@ public function load($tokenId, $params = [])
     {
         return $this->client()->get('tokens/{tokenId}', ['tokenId' => $tokenId] + (array) $params);
     }
+
+    /**
+     * @param string $tokenId
+     * @param array|ArrayObject $params
+     *
+     * @throws NotFoundException The resource data does exist
+     *
+     * @return PaymentCardToken
+     */
+    public function expire($tokenId, $params = [])
```

Comment:
Also, I see in API doc we do not need any request data. Or the documentation is not actual?